pkg/pillar/cmd/zedmanager/handlevolumemgr.go

```diff
@@ -60,6 +61,8 @@ func AddOrRefcountVolumeConfig(ctx *zedmanagerContext, blobSha256 string,
 		}
 		publishVolumeConfig(ctx, &n)
 	}
+	base.NewRelationObject(base.AddRelationType, base.AppInstanceConfigLogType, appInstID.String(),
+		base.VolumeLogType, key).Infof("App instance to volume relation ADD")
```

Comment:
The string seems completely redundant since the Add is already in AddRelationType

Reply:
makes sense. will remove.